test_scripts/API/SetGlobalProperties/026_ATF_UI_IsReady_no_resp_SetGlobalProp_UI_SetGlobalProp_UNSUP_RES.lua

```diff
@@ -0,0 +1,161 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one component of RPC
```

Comment:
@vvvakulenko, please update name of requirement "SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions"